lessons/introR/main.Rmd

```diff
@@ -0,0 +1,291 @@
+---
```

Comment:
Jekyll may try to interpret this front-matter, which may or may not cause errors when building the site. We'll have to see what happens.